### PR TITLE
Replace land mines

### PR DESCRIPTION
**Pull request by SiegeSpud**
Replaced land mines with updated ones based on Daniel Cook's artwork.

### CODE REVIEW

File: mods/hv/bits/mines.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FrameSize: 20,20
-FrameAmount: 4
-Offset: 0,0
-Author: Daniel Cook
-License: CC-BY-3.0-US
```
Comment (Mailaender) [inline mods/hv/bits/mines.yaml:5]: Looks like you introduced a line ending problem.

**Comment by Mailaender** [comment]
Again, you didn't bake in the metadata.

**Comment by Dzierzan** [comment]
Besides, they look so custom, you could easily remove Daniel from credits.

**Comment by Mailaender** [comment]
They are based on the 4th sprite.

**Pull request closed without merging**